cpp/include/cudf/column/column_view.hpp

```diff
@@ -176,9 +176,13 @@ class column_view_base {
    *
    * @param[in] begin The starting index of the range (inclusive).
    * @param[in] end The index of the last element in the range (exclusive).
+   * @param[in] stream CUDA stream used for device memory operations and kernel launches
    * @return The count of null elements in the given range
    */
-  [[nodiscard]] size_type null_count(size_type begin, size_type end) const;
+  [[nodiscard]] size_type null_count(
+    size_type begin,
+    size_type end,
+    rmm::cuda_stream_view stream = cudf::get_default_stream()) const;
```

Comment:
There's one usage of this function inside libcudf (rolling/grouped_rolling.cu) that I think needs now to plumb a stream through, perhaps an issue and assign it to me, since I am working in that code at the moment?